std_msgs/Header header
float64 scale
float64 minx
float64 miny
float64 maxx
float64 maxy
geometry_msgs/PoseArray[] goals